sdk/java-sdk/src/main/java/kalix/javasdk/workflowentity/WorkflowEntity.java

```diff
@@ -294,10 +333,29 @@ public StepBuilder(String name) {
       this.name = name;
     }
 
+    /**
+     * Build a step action with a call to an existing Kalix component via {@link DeferredCall}.
+     *
+     * @param callFactory Factory method for creating deferred call.
+     * @return Step builder.
```

Comment:
Just to be sure. It's a Step builder but the class name is a `CallBuilder`?

Reply:
Renamed